pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -17,7 +16,6 @@
   vscode-utils,
   zlib,
 }:
-
```

Comment:
What are you doing?

Reply:
Not my fault the file isn't formatted correctly. Just formatting it as per https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vscode/extensions/README.md?plain=1#L10-L11 🫡

Reply:
I have just executed `nixfmt default.nix` and it did not cause any issues. Those changes are unrelated to `nixfmt`.

Reply:
Who doesn't love Nix formatters